src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -136,6 +136,13 @@ private readonly Dictionary<RestoreTargetGraph, Dictionary<string, LibraryInclud
         private const string AuditSuppressedAdvisoriesTotalPackageDownloadWarningsSuppressedCount = "Audit.Vulnerability.PackageDownloads.TotalWarningsSuppressed.Count";
         private const string AuditSuppressedAdvisoriesDistinctPackageDownloadAdvisoriesSuppressedCount = "Audit.Vulnerability.PackageDownload.DistinctAdvisoriesSuppressed.Count";
 
+        // PackagePruning names
```

Comment:
nit: comment not necessary

Reply:
Probably unnecessary, but it's a pattern we have with all the telemetry properties, so I just kept it. 
To be honest, there's nearly 100 props, so I've found that helpful.

<img width="733" height="554" alt="image" src="https://github.com/user-attachments/assets/0cd5305e-11e6-4b11-85f4-65a1958cbb8a" />